### DISCUSSION TITLE
I tried, but not working

### DISCUSSION THREAD

**Discussion by t0linixer** [juddydev/react-3D#9] [Q&A] [answered]
I tried, but not working

**Comment by t0linixer** [comment]
you need to create co-authored commits on the command line(in a terminal way).

e.g.
![image](https://github.com/juddydev/react-3D/assets/167388906/f7eaf107-286b-42be-8101-a0bd5e1f4471)

One thing to note is that the closing quotation mark(") must be written
after the co-author's email, and the email must be enclosed in <>

(more details )
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Reply by alpine-storm** [reply, marked as answer]
thank you

**Comment by alpine-storm** [comment]
you need to create co-authored commits on the command line(in a terminal way).

e.g.
image

One thing to note is that the closing quotation mark(") must be written
after the co-author's email, and the email must be enclosed in <>

(more details )
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors
